build_keyboard.mk

```
@@ -299,6 +302,11 @@ ifneq ("$(wildcard $(KEYMAP_PATH)/config.h)","")
     CONFIG_H += $(KEYMAP_PATH)/config.h
 endif
 
+# Userspace post config.h pass
+ifneq ("$(wildcard $(USER_PATH)/config.h)","")
```

This should check for `post_config.h` instead of `config.h`.
```suggestion
ifneq ("$(wildcard $(USER_PATH)/post_config.h)","")
```